wpeverest/user-registration
# Tweak - Design tweak for Login and Registration Form Frontend.

### All Submissions:


### Changes proposed in this Pull Request:



Design tweak for login and registration form.

### How to test the changes in this Pull Request:

1. Test the Login and Registration functionality and verify if everything is working properly.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Design tweak for Login and Registration Form Frontend.
